lists/social.txt

```diff
@@ -42,6 +42,7 @@
 
 ! LinkedIn
 linkedin.com
+licdn.com
```

Comment:
This should probably go in the ads and trackers block list instead of the social one.

Reply:
Ok. No `||` or `^`?

Reply:
No no, you should move your Linkedin block to lists/ads.txt instead.